Lib/test/_test_embed_structseq.py

```diff
@@ -1,29 +1,28 @@
 import sys
 import types
-import unittest
 
 
 # bpo-46417: Test that structseq types used by the sys module are still
 # valid when Py_Finalize()/Py_Initialize() are called multiple times.
-class TestStructSeq(unittest.TestCase):
+class TestStructSeq:
     # test PyTypeObject members
-    def check_structseq(self, obj_type):
+    def _check_structseq(self, obj_type):
         # ob_refcnt
-        self.assertGreaterEqual(sys.getrefcount(obj_type), 1)
+        assert(sys.getrefcount(obj_type) > 1)
```

Comment:
Note that `assert` isn't a function but a statement, and the parentheses are not needed (and probably violating PEP 8 :-).

Reply:
Ah you are right... I've been way too much time in C. Fixed!

Reply:
That happens to me too. 😄